Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -82,24 +82,30 @@ size_t Sheet::add_row()
     return m_rows++;
 }
 
-static size_t convert_from_string(StringView str, unsigned base = 26, StringView map = {})
+static constexpr size_t convert_from_string(StringView str)
 {
-    if (map.is_null())
-        map = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-
-    VERIFY(base >= 2 && base <= map.length());
-
     size_t value = 0;
-    for (size_t i = str.length(); i > 0; --i) {
-        auto digit_value = map.find(str[i - 1]).value_or(0);
+    const auto len = str.length();
+    for (auto i = 0u; i < len; i++) {
+        assert(str[i] >= 'A' && str[i] <= 'Z');
+        size_t digit_value = str[i] - 'A';
         // NOTE: Refer to the note in `String::bijective_base_from()'.
-        if (i == str.length() && str.length() > 1)
-            ++digit_value;
-        value = value * base + digit_value;
+        if (i == 0 and len > 1)
+            digit_value++;
+        value += digit_value * AK::pow<float>(26, len - 1 - i);
```

Comment:
`std::pow()` is not templated and supports automatic type promotion for both arguments.
`AK::pow()` is templated with single parameter. 
This causes error here, as first and second parameter are not of same type.
I have supplied template parameter manually to work around it.

But I think `AK::pow()` can be made aligned with `std::pow`(two template parameters) .